sections/06_discussion.md

```diff
@@ -2,57 +2,78 @@
-usually necessitate an FDR of 5-25%. When predicted biochemical activities are
-used as features in other models, such as gene expression models, a low FDR may
-not be as critical if the downstream models can satisfy their evaluation
-criteria.  An FDR of 50% in this context may suffice.
-
-What is the correspondence between these metrics and commonly used
-classification metrics such as auPRC (area under the precision-recall curve) and
-auROC? auPRC evaluates the average precision, or equivalently, the average FDR
-across all recall thresholds. This metric provides an overall estimate of
-performance across all possible use cases, which can be misleading for targeted
-validation experiments. For example, classification of TF binding sites can
-exhibit a recall of 0% at 10% FDR and auPRC greater than 0.6. In this case, the
-auPRC may be competitive, but the predictions are ill-suited for targeted
-validation that can only examine a few of the highest-confidence predictions.
-Likewise, auROC evaluates the average recall across all false positive rate
-(FPR) thresholds, which is often a highly misleading metric in class-imbalanced
-domains [@doi:10.1145/1143844.1143874 @doi:10.1038/nmeth.3945]. For example,
-consider a classification model with recall of 0% at FDR less than 25% and 100%
-recall at FDR greater than 25%. In the context of TF binding predictions where
-only 1% of genomic regions are bound by the TF, this is equivalent to a recall
-of 100% for FPR greater than 0.33%. In other words, the auROC would be 0.9967,
-but the classifier would be useless for targeted validation. It is not unusual
-to obtain a chromosome-wide auROC greater than 0.99 for TF binding predictions
-but a recall of 0% at 10% FDR.
+Making predictions in the presence of high class imbalance and differences
+between training and generalization data is a common feature of many large
+biomedical datasets, including deep learning models of genomic features, patient
+classification, disease detection, and virtual screening. Prediction of
+transcription factor binding sites exemplifies the difficulties with learning
+from highly imbalanced data. The human genome has 3 billion base pairs, and only
+a small fraction of them are implicated in specific biochemical activities. Less
+than 1% of the genome can be confidently labeled as bound for most transcription
+factors.
+
+Estimating the false discovery rate (FDR) is a standard method of evaluation in
+genomics that can also be applied to deep learning model predictions of genomic
+features. Using deep learning predictions for targeted validation experiments of
+specific biochemical activities necessitates a more stringent FDR (typically
+5-25%). However, when predicted biochemical activities are used as features in
+other models, such as gene expression models, a low FDR may not be as critical
+(or even necessary) if the downstream models can satisfy their evaluation
+criteria.
```

Comment:
@avctrh: following up on the discussion, how about rewording last sentence into
```However, when predicted biochemical activities are used as features in other models, such as gene expression models, a low FDR may not be necessary.```
or maybe more explicitly state that metrics examining overall performance (auPRC/rank correlation/etc) would be more appropriate in that setting.